notebooks/2_PrincipalCovariatesRegression.ipynb

```diff
@@ -560,7 +572,7 @@
    "source": [
     "## Computing the Projectors\n",
     "\n",
-    "The similarity is maximized when the orthogonal matrix $\\mathbf{C}^{1/2}\\mathbf{P}_{X\\tilde{T}}$ matches the principal eigenvalues of $\\mathbf{\\tilde{C}}$, i.e. $\\mathbf{P}_{X\\tilde{T}} = \\mathbf{C}^{-1/2}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}$."
+    "The similarity is maximized when the orthonormal column vectors in matrix $\\mathbf{C}^{1/2}\\mathbf{P}_{X\\tilde{T}}$ match the principal eigenvalues of $\\mathbf{\\tilde{C}}$, i.e. $\\mathbf{P}_{X\\tilde{T}} = \\mathbf{C}^{-1/2}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}$. Similar to [PCA](1_LinearMethods.ipynb), we require that $\\mathbf{X}_{PcovR} = \\mathbf{\\tilde{T}}\\mathbf{P}_{\\tilde{T}X}$ perfectly approximates the portion of the feature space projected to the potential space, i.e., $\\mathbf{X}_{PcovR} \\mathbf{P}_{X\\tilde{T}} = \\mathbf{\\tilde{T}} $, which implies that $\\mathbf{P}_{\\tilde{T}X}\\mathbf{P}_{X\\tilde{T}}=\\mathbf{I}$ and $\\mathbf{P}_{\\tilde{T}X}=\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}^T \\mathbf{C}^{1/2}$"
```

Comment:
```suggestion
    "The similarity is maximized when the orthonormal column vectors in matrix $\\mathbf{C}^{1/2}\\mathbf{P}_{X\\tilde{T}}$ match the principal eigenvalues of $\\mathbf{\\tilde{C}}$, i.e. $\\mathbf{P}_{X\\tilde{T}} = \\mathbf{C}^{-1/2}\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}$. Similar to [PCA](1_LinearMethods.ipynb), we require that $\\mathbf{X}_{PcovR} = \\mathbf{\\tilde{T}}\\mathbf{P}_{\\tilde{T}X}$ approximates the portion of the feature space projected to the potential space, i.e., $\\mathbf{X}_{PcovR} \\mathbf{P}_{X\\tilde{T}} = \\mathbf{\\tilde{T}} $, which implies that $\\mathbf{P}_{\\tilde{T}X}\\mathbf{P}_{X\\tilde{T}}=\\mathbf{I}$ and $\\mathbf{P}_{\\tilde{T}X}=\\hat{\\mathbf{U}}_{\\mathbf{\\tilde{C}}}^T \\mathbf{C}^{1/2}$"
```

Reply:
Why shouldn't we use "perfectly" here? In the first notebook, we used "perfect approximation".
![image](https://user-images.githubusercontent.com/56018055/97762218-20dd6280-1b08-11eb-9f9f-f7e8231f82f1.png)

Reply:
That doesn't necessitate it be used here.